go.mod

```diff
@@ -3,18 +3,18 @@ module github.com/lightninglabs/aperture
 go 1.15
 
 require (
-	github.com/btcsuite/btcd v0.21.0-beta.0.20201208033208-6bd4c64a54fa
+	github.com/btcsuite/btcd v0.21.0-beta.0.20210513141527-ee5896bad5be
```

Comment:
We should be able to update our etcd client dependency as well and remove the replace workaround.

Reply:
Good point, I updated `lnd` to the version that bumps the `etcd` client version. Nice to finally get that replace removed.